specifyweb/frontend/js_src/lib/components/Header/Notifications.tsx

```diff
@@ -99,23 +120,25 @@ export function Notifications({
                   type,
                   payload: rest as IR<LocalizedString>,
                 })
+              ),
+            ].sort(
```

Comment:
(optional) the `// Make most recent notification first` comment was lost in the process. not a big deal, just a FYI in case that wasn't intentional